tests/test_deps.py

```
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
```

Ruff (0.11.9)

Shebang is present but file is not executable

(EXE001)